src/fonts.rs

```diff
@@ -17,8 +18,13 @@ pub use ffi::enums::{
     FontType,
     FontWeight,
     FontSlant,
+
```

Comment:
It seemed like the Font types were being grouped together so I kept TextClusterFlags separate to them. I can remove the empty line if you wish?